php-config/Emergence/People/RegistrationRequestHandler.config.php

```diff
@@ -1,3 +1,3 @@
 <?php
 
-Emergence\People\RegistrationRequestHandler::$enableRegistration = false;
\ No newline at end of file
+Emergence\People\RegistrationRequestHandler::$enableRegistration = true;
```

Comment:
do a rebase and edit this change out of whichever commit it is in

Reply:
Oops. I tried not to let that through, and failed

Reply:
✨